sample_apps/php/main.php

```diff
@@ -0,0 +1,109 @@
+<?php
+    require '/path/to/vendor/autoload.php';
+
+    const REGION = 'us-east-1';
+
+    const CLIENT_KEY = 'CLIENT_KEY_OVERRIDE';
+    const CLIENT_SECRET = 'CLIENT_SECRET_OVERRIDE';
+
+    const DB_NAME = 'TEST_PHP';
+    const DB_TABLE_NAME = 'TEST_TABLE_PHP';
+    const MAGNETIC_STORE_RETENTION_PERIOD_IN_DAYS = 1;
+    const MEMORY_STORE_RETENTION_PERIOD_IN_HOURS = 1;
+
+    function createDatabase($client, $dbName) {
+        return $client->createDatabase([
+                   'DatabaseName' => $dbName
+               ]);
+    }
+
+    function deleteDatabase($client, $dbName) {
+        return $client->deleteDatabase([
+                   'DatabaseName' => $dbName,
+               ]);
+    }
+
+    function createTable($client, $dbName, $dbTableName) {
+        return $client->createTable([
+                    'DatabaseName' => $dbName,
+                    'RetentionProperties' => [
+                       'MagneticStoreRetentionPeriodInDays' => MAGNETIC_STORE_RETENTION_PERIOD_IN_DAYS,
+                       'MemoryStoreRetentionPeriodInHours' => MEMORY_STORE_RETENTION_PERIOD_IN_HOURS,
+                    ],
+                    'TableName' => $dbTableName
+                ]);
+    }
+
+    function deleteTable($client, $dbName, $dbTableName) {
+        return $client->deleteTable([
+                   'DatabaseName' => $dbName,
+                   'TableName' => $dbTableName,
+               ]);
+    }
+
+    function writeRecords($client, $dbName, $dbTableName, $records) {
```

Comment:
Can you add rejected records exception handling? Sample snippet provided below

```
if (isset($response['RejectedRecords']) && !empty($response['RejectedRecords'])) {
        echo "Some records were rejected:\n";
        foreach ($response['RejectedRecords'] as $rejectedRecord) {
            echo "Reason: " . $rejectedRecord['Reason'] . "\n";
            echo "ExistingRecord: " . json_encode($rejectedRecord['ExistingRecord']) . "\n";
        }
    } else {
        echo "Data inserted successfully!\n";
    }
```

Reply:
Sample has been added, thank you for your comment